cardinal/system/t_tokeniser_system_test.go

```diff
@@ -0,0 +1,81 @@
+package system
+
+import (
+	"testing"
+
+	"github.com/ArchetypalTech/TheOrugginTrail-ArgusWE/cardinal/enums"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTokeniserSystem(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	assert.NotNil(t, ts.cmdLookup)
+	assert.NotNil(t, ts.dirLookup)
+	assert.NotNil(t, ts.dirObjLookup)
+	assert.NotNil(t, ts.objLookup)
+	assert.NotNil(t, ts.grammarLookup)
+	assert.NotNil(t, ts.responseLookup)
+
+	// Check if all predefined values are set correctly
+	assert.Equal(t, enums.ActionTypeGo, ts.cmdLookup["GO"])
+	assert.Equal(t, enums.ObjectTypeFootball, ts.objLookup["FOOTBALL"])
+	assert.Equal(t, enums.DirectionTypeNorth, ts.dirLookup["NORTH"])
+	assert.Equal(t, enums.DirObjectTypeDoor, ts.dirObjLookup["DOOR"])
+	assert.Equal(t, enums.GrammarTypeDefinitionArticle, ts.grammarLookup["THE"])
+	assert.ElementsMatch(t, []enums.ActionType{enums.ActionTypeBreak, enums.ActionTypeHit, enums.ActionTypeDamage}, ts.responseLookup[enums.ActionTypeKick])
+}
+
+func TestLookupFunctions(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	assert.Equal(t, enums.ObjectTypeFootball, ts.GetObjectType("FOOTBALL"))
+	assert.Equal(t, enums.ActionTypeGo, ts.GetActionType("GO"))
+	assert.Equal(t, enums.GrammarTypeDefinitionArticle, ts.GetGrammarType("THE"))
+	assert.Equal(t, enums.DirectionTypeNorth, ts.GetDirectionType("NORTH"))
+}
+
+func TestFishTokensKickTheBallToTheWindow(t *testing.T) {
+	ts := NewTokeniserSystem()
+
+	tokens := []string{"KICK", "THE", "BALL", "TO", "THE", "WINDOW"}
```

Comment:
we dont need the Object type. we Need the VRB the DOBJ and the IOBJ if it exists

Reply:
to put another way this should be VRB:kick, DOBJ:football, IOBJ:window there is no object for the tokeniser, ever only the 3 mentioned things, (vrb, dobj, iobj)